decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
```

Comment:
- Hey, kindly make the nameable accessible to it value can be read as it's meant to be a getter variable.
`  attr_accessor :nameable`